wiki/Glossary/fr.md

```diff
@@ -0,0 +1,999 @@
+Si le joueur obtenait un combo de 500, la couleur du combo fire passait du jaune au bleu.
+Le combo fire s'éteignait de lui-même si le combo était cassé, mais pouvait toujours être redéclanché.
+
+Cette fonctionnalité a été par la suite désactivée à cause de problèmes de performances.
+
+### Cookie
+
+![](img/Intro_static.jpg "The cookie is typically used to describe the circular osu!logo which appears at the very start of the game, and refers to its shape.")
+
+Le cookie est un autre terme désignant le "osu!logo", qui apparaît plusieurs fois dans le client osu!.
+Le cookie est présent dans le menu principal, en mode solo, et précédemment dans les vieux rapports de crash ([voir un exemple](img/Pippi_corruption.jpg)).
+
+## D
+
+### Deathstreams
+
+Un deathstream est un long flux ou les objets se suivent les uns les autres en se succédant de manière rapide et/ou en formant des patterns assez confus.
+Ils amènent généralement le joueur à perdre sa "stamina" de manière rapide, et potentiellement à échouer la beatmap.
+
+Il est fortement suggéré d'utiliser les deathstreams uniquement lorsque la beatmap l'insinue et/ou qu'elle a un BPM élevé.
+Ce principe **ne devrait pas** être utilisé pour faire des overmaps!
+
+### Etoiles (terme original : Difficulty stars)
+
+Les étoiles sont la représentation visuelle de la difficulté d'une beatmap.
+Cette note est comprise entre 0 et 10 étoiles (en théorie), et est basé sur un algorithme.
+
+osu! utilise actuellement deux algorithmes pour calculer les étoiles:
+
+- l'algorithme de peppy
+  - déterminé en utilisant le multiplicateur bonus de score (en fonction de la difficulté) et l'algorithme d'eyup
+- l'algorithme d'Echo (non utilisé actuellement)
+
+Vous pouvez voir les étoiles d'une beatmap en la testant dans l'éditeur, sur l'écran de sélection de la chanson, ou sur son listing sur le site.
+
+### Difficultés
+
+Les beatmaps peuvent avoir plusieurs difficultés, allant de facile (Easy) à Expert et plus.
+Une beatmap devrait généralement avoir entre deux et cinq versoins de difficultés variables.
+Cela permettrait aux joueurs novices et expérimentés de pouvoir jouer et apprécier la beatmap.
+
+Quelques-uns des facteurs qui peuvent affecter la difficulté d'une version sont le nombre d'objets dans la beatmap, qu'importe que le rythme choisi pour le placement des objets soie le rythme principal ou un rythme d'arrière plan plus difficile à entendre,la longueur des combos, la valeur d'espacement des pulsations, le taux de drainage, la limite de précision requise par le jeu pour enregistrer une frappe ,la taille du cercle de frappe, etc..
+Les étoiles de difficulté sont un moyen de jauger la version de difficulté d'un niveau.
+Le "osu!website" trie les difficultés en 5 catégories (en anglais):
+
+- Easy,
+- Normal,
+- Hard,
+- Insane, and
+- Expert
+
+### Temps de drainage (terme original : drain time) <!--Review needed | max priority -->
```

Comment:
ditto

if you are going to include the original term, please put it underneath instead

```
### Temps de drainage

_Terme original : drain time_
```